tests/common/test_command_populate_aip_checksums.py

```diff
@@ -0,0 +1,262 @@
+    )
+    algo = "sha256"
+    assert captured.out.splitlines() == [
+        f"AIP {compressed_package.uuid} updated with {algo} checksum {compressed_package_checksum}",
+        f"Complete. Checksums added for all {models.Package.objects.count()} identified AIPs.",
+    ]
+
+
+@pytest.mark.django_db
+@mock.patch("archivematica.storage_service.common.utils.generate_checksum")
+def test_command_updates_checksum_for_uncompressed_aip(
+    generate_checksum: mock.Mock,
+    capsys: pytest.CaptureFixture[str],
+    uncompressed_package: models.Package,
+    aip_storage_fs_location: models.Location,
+) -> None:
+    uncompressed_package_checksum = (
+        "c2924159fcbbeadf8d7f3962b43ec1bf301e1b4f12dd28a8b89ec819f3714848"
+    )
+    generate_checksum.return_value = mock.Mock(
+        **{
+            "hexdigest.return_value": uncompressed_package_checksum,
+        }
+    )
+
+    call_command(
+        "populate_aip_checksums",
+        "--location-uuid",
+        aip_storage_fs_location.uuid,
+    )
+
+    captured = capsys.readouterr()
+    assert captured.out.splitlines() == [
+        f"AIP {uncompressed_package.uuid} updated with sha256 checksum {uncompressed_package_checksum}",
+        f"Complete. Checksums added for all {models.Package.objects.count()} identified AIPs.",
+    ]
+
+
+@pytest.mark.django_db
+@mock.patch("archivematica.storage_service.common.utils.generate_checksum")
+def test_command_updates_checksum_for_download_aips(
+    generate_checksum: mock.Mock,
+    capsys: pytest.CaptureFixture[str],
+    compressed_package: models.Package,
+    uncompressed_package: models.Package,
+    aip_storage_fs_location: models.Location,
+) -> None:
+    uncompressed_package_checksum = (
+        "c2924159fcbbeadf8d7f3962b43ec1bf301e1b4f12dd28a8b89ec819f3714848"
+    )
+    generate_checksum.return_value = mock.Mock(
+        **{
+            "hexdigest.return_value": uncompressed_package_checksum,
+        }
+    )
+
+    with mock.patch.object(models.Package, "full_pointer_file_path", POINTER_FILE_PATH):
+        call_command(
+            "populate_aip_checksums",
+            "--location-uuid",
+            aip_storage_fs_location.uuid,
+            "--download",
+        )
+
+    captured = capsys.readouterr()
+    compressed_package_checksum = (
+        "c2924159fcbbeadf8d7f3962b43ec1bf301e1b4f12dd28a8b89ec819f3714747"
+    )
+    algo = "sha256"
+    assert captured.out.splitlines() == [
+        f"AIP {compressed_package.uuid} updated with {algo} checksum {compressed_package_checksum}",
+        f"AIP {uncompressed_package.uuid} updated with {algo} checksum {uncompressed_package_checksum}",
+        f"Complete. Checksums added for all {models.Package.objects.count()} identified AIPs.",
+    ]
+
+
+@pytest.mark.django_db
+@mock.patch("archivematica.storage_service.common.utils.generate_checksum")
+def test_command_fails_when_checksum_is_missing_for_uncompressed_aips(
```

Comment:
```suggestion
def test_command_fails_when_checksum_is_missing_for_uncompressed_aip(
```

To maintain consistency with the test above:

```python
def test_command_fails_when_checksum_is_missing_for_compressed_aip(
```